subprojects/docs/src/snippets/userguide/dependencyManagement/workingWithDependencies/iterateDependencies/groovy/build.gradle

```diff
@@ -13,11 +13,12 @@ dependencies {
 
 // tag::iteration-task[]
 task iterateDeclaredDependencies {
+    def dependencyCoordinates = provider {
```

Comment:
⭕️ I think these changes to the userguide deserve some comment explaining why there's a provider there now instead of the plain imperative version that was before OR we should refrain from touching userguide samples at this time.

Reply:
Good point. I left the user guide samples untouched.